macros/get_advertiser_history_columns.sql

```diff
@@ -2,16 +2,13 @@
 
 {% set columns = [
     {"name": "_fivetran_synced", "datatype": dbt.type_timestamp()},
-    {"name": "billing_profile_status", "datatype": dbt.type_string()},
-    {"name": "billing_type", "datatype": dbt.type_string()},
     {"name": "country", "datatype": dbt.type_string()},
     {"name": "created_time", "datatype": dbt.type_timestamp()},
     {"name": "currency", "datatype": dbt.type_string()},
     {"name": "id", "datatype": dbt.type_string()},
-    {"name": "merchant_id", "datatype": dbt.type_string()},
+    {"name": "owner_username", "datatype": dbt.type_string()},
     {"name": "name", "datatype": dbt.type_string()},
-    {"name": "owner_user_id", "datatype": dbt.type_string()},
```

Comment:
Ahh... I was going off the list in our ticket since the release notes weren't available when I made the edits and our PM confirmed, and owner_user_id was included in there. But now I see that it's not in the release notes as a deprecated field, so I'll add it back in